CMakeLists.txt

```
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(easy_aruco)
 
-find_package(catkin REQUIRED COMPONENTS sensor_msgs cv_bridge tf2_ros tf2_geometry_msgs)
+find_package(catkin REQUIRED COMPONENTS sensor_msgs cv_bridge tf2_ros tf2_geometry_msgs visualization_msgs)
```

```suggestion
find_package(catkin REQUIRED COMPONENTS sensor_msgs cv_bridge tf2_ros tf2_geometry_msgs)
```
This looks unnecessary